zkevm-circuits/src/evm_circuit/execution/calldataload.rs

```diff
@@ -503,4 +507,27 @@ mod test {
             Some(10u64),
         );
     }
+
+    fn test_busmapping(offset: Word) {
+        let bytecode = bytecode! {
+            PUSH32(offset)
+            CALLDATALOAD
+            STOP
+        };
```

Comment:
This is still only testing `root_call` situations.

Tests for `non_root_call` scenarios are required.